spring-cloud-gcp-data-spanner/src/test/java/org/springframework/cloud/gcp/data/spanner/core/convert/KeyConversionTests.java

```diff
@@ -88,11 +86,12 @@ public KeyConversionTests(String testCaseName, Object objectToTest, Object expec
 	public void keyWritingTestCase() {
 		try {
 			Key key = this.spannerEntityWriter.convertToKey(this.objectToTest);
-			assertThat(key, is(this.expectedResult));
+			assertThat(key).isEqualTo(this.expectedResult);
 		}
 		catch (Exception e) {
```

Comment:
This one appears I can't quite change since it was written as Parameterized test; looks like both the throw and no-throw branches need to be in the code.